Tests/Tracking/APITest.swift

```diff
@@ -73,6 +77,9 @@ class TrackingAPITest: UnitTest {
         mock.screen(name: "", data: encodableData)
         instance.screen(name: "", data: encodableData)
 
+        checkDeviceProfileAttributes()
+    }
+    func checkDeviceProfileAttributes() {
```

Comment:
dont see this being reused anywhere else so any reason to have this be its own function rather than remain inline in the test

Reply:
Lint shows error as the number of lines in function exceed so modulated the code into functions